inc/assets.php

```diff
@@ -48,7 +49,7 @@ function( $carry, $port ) {
 
 	$headers['Content-Security-Policy'] = preg_replace(
 		"/script-src 'self'/",
-		"script-src 'self' blob: https://wikimedia.vipdev.lndo.site",
+		"script-src 'self' blob: https://wikimedia.vipdev.lndo.site https://wikimediafoundation-org-develop.go-vip.co",
```

Comment:
Can we add this to the security integration plugin in the Foundation site project instead? It is not specific to this plugin, technically

Reply:
Sure, I'll add a ticket to add that in and remove it from here 👍, I'll leave it in now so that we can demo.